eventlogger/hapi_server.js

```diff
@@ -1,16 +1,25 @@
 var requests_start_time = {};
+var assign = require('lodash').assign;
+
+module.exports.watch = function  (logger, server, options) {
+  options = assign({}, {
+    ignorePaths: []
+  }, options);
+
+  var ignorePaths = options.ignorePaths;
 
-module.exports.watch = function  (logger, server) {
   server
   .on('request-internal', function (request, data, tags) {
-    if (tags.received && request.path !== '/api/v2/test') {
+    if (tags.received && request.path !== '/api/v2/test' &&
```

Comment:
I could put this on defaults ignored patch but I vote for removing it and using the configuration option on api2 or leaving this this way because it is much more explicit OMHO